src/otb.mk

```diff
@@ -0,0 +1,57 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := otb
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 4.2.1
+$(PKG)_CHECKSUM := c4f1299a2828a6f6acb81c1e022c706b7b7f10ea
+$(PKG)_SUBDIR   := OTB-$($(PKG)_VERSION)
+$(PKG)_FILE     := $($(PKG)_SUBDIR).tgz
+$(PKG)_URL      := http://orfeo-toolbox.org/packages/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc gdal expat itk ossim opencv boost curl qt
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'https://sourceforge.net/projects/itk/files/itk/' | \
+    $(SED) -n 's,.*/\([0-9][^"]*\)/".*,\1,p' | \
+    head -1
+endef
+
+define $(PKG)_BUILD
+    mkdir '$(1).build'
+#copy otb.conf to build directory.
+    cp '$(1)/otb.conf' '$(1).build/otb.conf'
+    cd '$(1).build' && cmake \
+        -DCMAKE_TOOLCHAIN_FILE='$(CMAKE_TOOLCHAIN_FILE)' \
+    $(if $(BUILD_SHARED),\
+        -DBUILD_SHARED_LIBS=TRUE ) \
+    -DBUILD_TESTING=FALSE \
+    -DBUILD_EXAMPLES=FALSE \
+    -DBUILD_APPLICATIONS=TRUE \
```

Comment:
these are shared libs loaded by otbApplicationLauncherCommandLine. for the ease there are .sh and .bat scripts for every application